tests/client/test_base_client.py

```diff
@@ -538,7 +538,9 @@ async def mocked_client(
     responses: List[Response],
     **client_kwargs: Dict[str, Any],
 ) -> AsyncGenerator[Tuple[PrefectHttpxAsyncClient, mock.AsyncMock], None]:
-    with mock.patch("httpx.AsyncClient.send", autospec=True) as send:
+    with mock.patch(
+        "prefect.client.base.httpx.AsyncClient.send", autospec=True
+    ) as send:
```

Comment:
I don't think this is materially different?  When you patch a class's instance method, you're going to end up doing it for all instances process-wide